### ISSUE TITLE
[BUG]: In development, there is a button on my page that can be triggered

### ISSUE THREAD

**Issue by hexluxiang** [paypal/react-paypal-js#435]
### Is there an existing issue for this?

- [X] I have searched the existing issues.

### 🐞 Describe the Bug

In development, there is a button on my page that can be triggered vault = true，this is PayPalScriptProvider config。When the browser is set to language 'en_US' and vault = true，cant not use paypal debit or credit card payfor popu，but when vault=false, payment can be used。Or set locale = 'zh_CN', it is ok。

### 😕 Current Behavior

https://www.sandbox.paypal.com/smart/card-fields?sessionID=uid_7cfb6c2706_mti6mzm6mze&buttonSessionID=uid_98f0b8ecfa_mti6mzy6mty&locale.x=en_US&commit=true&env=sandbox&sdkMeta=xxxxxxxx&disable-card=&token=9UF4309477283115J

400 (Bad Request)

### 🤔 Expected Behavior

No matter what language is set in my browser, the window will be able to initiate credit card payments

### 🔬 Minimal Reproduction

```markdown
1. Set the browser to English ('en_US')
2. Do not configure the local configuration of PayPalButton
3. Cancel automatic renewal (vault = false)
4. Unable to initiate credit card payment
```

### 🌍 Environment

```markdown
| Software         | Version(s) |
| ---------------- | ---------- |
| react-paypal-js  |  7.8.1     |
| Browser          | 124.0.6367.208   |
| Operating System | Win10 |
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### ➕ Anything else?

https://paypal.github.io/react-paypal-js/?path=/docs/example-paypalbuttons--default

**Comment by wsbrunson** [comment]
That's correct, there are certain payment methods like PayPal Credit that don't work with `vault=true`. Also you need to turn on the advanced vault settings for your merchant in the developer dashboard to get access to vaulting other payment methods like the Credit Card button

**Comment by vimalvataliya1904** [comment]
I am also facing the same issue..
https://www.paypal-community.com/t5/PayPal-Payments-Standard/Using-Paypal-Smart-Buttons-for-checkout-sometimes-fails-with/m-p/3153618

any solution for this 400 error while click on pay with credit card button?

**Comment by hexluxiang** [comment]
> That's correct, there are certain payment methods like PayPal Credit that don't work with `vault=true`. Also you need to turn on the advanced vault settings for your merchant in the developer dashboard to get access to vaulting other payment methods like the Credit Card button没错，某些付款方式（如 PayPal Credit）不适用于 `vault=true` .此外，您还需要在开发者仪表板中为商家打开高级保管库设置，才能访问保管库其他付款方式，例如信用卡按钮

**Comment by hexluxiang** [comment]
> That's correct, there are certain payment methods like PayPal Credit that don't work with `vault=true`. Also you need to turn on the advanced vault settings for your merchant in the developer dashboard to get access to vaulting other payment methods like the Credit Card button

How to open the advanced repository settings for merchants in the developer dashboard as you mentioned? Can you teach me? Or give me some screenshot tips, etc.  !Thanks

**Comment by wsbrunson** [comment]
We have moved this repository to our paypal-js monorepo: https://github.com/paypal/paypal-js

We will be archiving this repo at the end of the day which will turn off Issues activity. If this is still an issue for you, please re-open this issue here: https://github.com/paypal/paypal-js/issues
